src/eslint-config-adeira/__tests__/__snapshots__/presets.test.js.snap

```diff
@@ -571,6 +571,7 @@ Object {
   "plugins": Array [
     "eslint-plugin-flowtype",
     "eslint-plugin-fb-flow",
+    "eslint-plugin-prettier",
```

Comment:
This was somehow strangely broken in the previous presets: the prettier rules are being used but the plugin was not specified anywhere.